music_visualizer/graphics/platform.py

```diff
@@ -0,0 +1,93 @@
+## Imports
+import pygame
+import sys
+
+"""
+    This is a simple platform class for respresenting a platform 
+    in the video
+"""
+class Platform(object):
+    """! Class with all the implementation for the platform
+    on the visualizer.
+
+    This class contains all the platform properties and the
+    methods to update the platform's visual
+    """
+
+    def __init__ (self, screen, length, width, color, x, y):
```

Comment:
```suggestion
    def __init__ (self, screen, width, height, color, x, y):
```
Talvez essa nomenclatura seja melhor. Pelo menos eu penso em Width como horizontal.